tools/notification-configuration/github-codeowner-subscriber/Program.cs

```diff
@@ -47,14 +47,18 @@ class Program
             }))
 #pragma warning restore CS0618 // Type or member is obsolete
             {
-                var devOpsService = AzureDevOpsService.CreateAzureDevOpsService(devOpsTokenVar,
+                var devOpsService = AzureDevOpsService.CreateAzureDevOpsService(
+                    Environment.GetEnvironmentVariable(devOpsTokenVar),
                     $"https://dev.azure.com/{organization}/",
                     loggerFactory.CreateLogger<AzureDevOpsService>()
                 );
                 
                 var gitHubServiceLogger = loggerFactory.CreateLogger<GitHubService>();
                 var gitHubService = new GitHubService(gitHubServiceLogger);
-                var credential = new ClientSecretCredential(aadTenantVar, aadAppIdVar, aadAppSecretVar);
+                var credential = new ClientSecretCredential(
+                    Environment.GetEnvironmentVariable(aadTenantVar), 
+                    Environment.GetEnvironmentVariable(aadAppIdVar), 
+                    Environment.GetEnvironmentVariable(aadAppSecretVar));
```

Comment:
Did these calls just not work before?

Reply:
This is what wes suggested. Env var is much safer than passing secret value.
He suggested we converted to env var